phase1-coordinator/src/objects/round.rs

```diff
@@ -794,19 +818,19 @@ impl Round {
     pub(crate) fn remove_locks_unsafe(
         &mut self,
         storage: &mut StorageLock,
-        justification: &Justification,
+        drop: &DropParticipant,
     ) -> Result<(), CoordinatorError> {
         // Check that the justification is valid for this operation, and fetch the necessary state.
-        let (participant, locked_chunks) = match justification {
-            Justification::BanCurrent(data) => (&data.participant, &data.locked_chunks),
-            Justification::DropCurrent(data) => (&data.participant, &data.locked_chunks),
+        let (participant, locked_chunks) = match drop {
+            DropParticipant::BanCurrent(data) => (&data.participant, &data.locked_chunks),
+            DropParticipant::DropCurrent(data) => (&data.participant, &data.locked_chunks),
             _ => return Err(CoordinatorError::JustificationInvalid),
         };
 
         // Sanity check that the participant holds the lock for each specified chunk.
         let locked_chunks: Vec<_> = locked_chunks
             .par_iter()
-            .filter(|chunk_id| self.is_chunk_locked_by(**chunk_id, participant))
+            // .filter(|chunk_id| self.is_chunk_locked_by(**chunk_id, participant))
```

Comment:
This change is involved in the actual bugfix, it also needs to be confirmed whether this line can be removed